components/ToolbarNotifications/ToolbarNotifications.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { User } from '~/redux/ducks/user'
+import {
+  DropdownMenu,
+  DropdownToggler,
+  DropdownWithContext,
+} from '~/components/Dropdown'
+import ViewerNotifications from './ViewerNotifications'
+import Icon from '~/components/Icon'
+
+const Menu: React.FC<{ className?: string }> = styled(DropdownMenu)`
+  left: auto;
+  width: 500px;
+  min-height: 500px;
+`
+interface Notification {
+  id: number
+  read: boolean
+  type: string
+  project: string
+  org: string
+  image: {
+    image_medium_url: string
+  }
+  created_at: string
+}
+
+interface ToolbarNotificationsProps {
+  readonly className?: string
+  readonly viewer: User
+  readonly theme?: 'dark' | 'light'
+}
+
+const notificationsFake = [
+  {
+    id: 1,
+    read: true,
+    type: 'Confirm',
+    project: 'Nome da Vaga',
+    org: 'Nome da ONG',
+    image: {
+      image_medium_url:
+        'https://storage.googleapis.com/atados-v3/user-uploaded/images-medium/e7b7bce0-e287-4ef3-a3f7-d8904097e4af.jpeg',
+    },
+    created_at: 'Agora',
+  },
+  {
+    id: 2,
+    read: false,
+    type: 'Delete',
+    project: 'Nome da Vaga',
+    org: 'Nome da ONG',
+    image: {
+      image_medium_url:
+        'https://storage.googleapis.com/atados-v3/user-uploaded/images-medium/e7b7bce0-e287-4ef3-a3f7-d8904097e4af.jpeg',
+    },
+    created_at: 'Agora',
+  },
+  {
+    id: 3,
+    read: false,
+    type: 'Alert',
+    project: 'Nome da Vaga',
+    org: 'Nome da ONG',
+    image: {
+      image_medium_url:
+        'https://storage.googleapis.com/atados-v3/user-uploaded/images-medium/e7b7bce0-e287-4ef3-a3f7-d8904097e4af.jpeg',
+    },
+    created_at: 'Agora',
+  },
+  {
+    id: 4,
+    read: true,
+    type: 'Warn',
+    project: 'Nome da Vaga',
+    org: 'Nome da ONG',
+    image: {
+      image_medium_url:
+        'https://storage.googleapis.com/atados-v3/user-uploaded/images-medium/e7b7bce0-e287-4ef3-a3f7-d8904097e4af.jpeg',
+    },
+    created_at: 'Agora',
+  },
+]
+
+const ToolbarNotifications: React.FC<ToolbarNotificationsProps> = ({
+  theme,
+}) => {
+  const [notifications, setNotifications] = useState<Notification[]>(
+    notificationsFake,
+  )
+  const [open, setOpen] = useState(false)
+  const numberNotifications = notifications?.filter(item => item.read === false)
```

Comment:
```suggestion
  const newNotificationsCount = notifications?.filter(item => item.read === false)
```

Muda o nome dessa variavel para refletir o que ela é. `numberNotifications` significa "número de notificações", mas na verdade aqui é o numero de notificações não lidas